packages/launchpad/src/checklist-item/style.scss

```
@@ -15,10 +15,23 @@
 	width: 100%;
 	margin: 0;
 	color: var(--color-neutral-100);
+	height: auto;
+	cursor: pointer;
+	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
```

This should use the `$sans` variable, which should already be imported via `@import "@automattic/typography/styles/variables";`.
```suggestion
	font-family: $sans;
```